appinventor/components/src/com/google/appinventor/components/runtime/util/ViewUtil.java

```diff
@@ -189,9 +196,39 @@ public static void setBackgroundImage(View view, Drawable drawable) {
   /**
    * Sets the image for an ImageView.
    */
-  public static void setImage(ImageView view, Drawable drawable) {
-    view.setImageDrawable(drawable);
-    if (drawable != null) {
+  public static void setImage(ComponentContainer container, ImageView view, String picturePath) {
+    GlideDrawableImageViewTarget imageViewTarget = new GlideDrawableImageViewTarget(view);
```

Comment:
I think we should only be using this logic if the file format is GIF. Otherwise, it puts Glide on the critical path for every image, which is not what I think we want.

Reply:
Also, what is the minSdk for Glide? We will need to note this in documentation.

Reply:
@ewpatton Glide requires a minimum SDK version of 10.